.vscode/launch.json

```diff
@@ -11,6 +11,7 @@
             "module": "uvicorn",
             "args": [
                 "nsls2api.main:app",
+                "--host=0.0.0.0",
```

Comment:
For dev?

Reply:
Yes, for WSL I needed to add this in order to access the site from Windoze itself.

Reply:
Not used in deployments so I just left it in here for convenience